python/scannerpy/table.py

```diff
@@ -37,13 +37,14 @@ def columns(self, index=None):
         return columns[index] if index is not None else columns
 
     def num_rows(self):
-        return self._descriptor.num_rows
+        return self._descriptor.end_rows[-1]
 
     def rows(self):
         if self._job is None:
             return list(range(self.num_rows()))
         else:
             if len(self._task.samples) == 1:
+                assert(False)
```

Comment:
Yeah, seems like it. Looks like it's the else clause here that should be `assert(False)`.